### PR TITLE
1.0.0 release

### PR DESCRIPTION
**Pull request by pandafy**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,13 +1,47 @@
...
+- Added support for formatting CSS and JS in `openwisp-qa-format <https://github.com/openwisp/openwisp-utils#openwisp-qa-format>`_
+- Added `git pre-push hook <https://github.com/openwisp/openwisp-utils/issues/161>`_
+
 Changes
 ~~~~~~~
 
-- Updated Django REST Framework to 3.12.x
+- `Updated OpenWISP's admin theme <https://medium.com/@niteshsinha1707/new-navigation-menu-and-ui-ux-improvements-project-report-a94c37514b7d>`_
+- Added support for Django 3.2.x and 4.0.x
+- Added support for Python 3.9
```
Comment (pandafy) [inline CHANGES.rst:28]: I am not sure if these should go under features or dependencies

Comment (nemesisdesign) [reply CHANGES.rst:28]: in dependencies

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Please fix the commit message as well.

File: CHANGES.rst [hunk 2]
```diff
@@ -1,13 +1,47 @@
...
+
+Dependencies
+~~~~~~~~~~~~
+
+- Bumped ``django-model-utils~=4.2.0``
+- Bumped ``black<=21.10b0``
+- Bumped ``djangorestframework~=3.13.0``
+- Added ``swapper~=1.3.0`` and ``django-compress-staticfiles~=1.0.1b``
+- Added support for Django ``3.2.x`` and ``4.0.x``
+- Added support for Python ``3.9``
```
Comment (nemesisdesign) [inline CHANGES.rst:36]: I think we should mention celery.

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**